packages/google_sign_in/google_sign_in/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 5.1.0
```

Comment:
Is this worth a minor bump, or should I make it a patch 5.0.7?

Reply:
Per other PR, 5.0.7.

Reply:
Done.